pyk/src/tests/integration/proof/test_refute_node.py

```diff
@@ -372,7 +373,7 @@ def test_apr_proof_refute_node_multiple_constraints(
         prover.advance_proof(proof, max_iterations=4)
         # After the minimization, nodes 7-10 created by the advancement of the proof
         # will have multiple constraints in their immediately preceding splits
-        proof.kcfg.minimize()
+        KCFGMinimizer(proof.kcfg).minimize()
```

Comment:
Here as well. Let's keep `KCFGMinimizer` private to the `kcfg` module.

Reply:
Sure, I'll change that!

Reply:
Done! Nothing changed in [test_refute_node.py](https://github.com/runtimeverification/k/pull/4607/files#diff-c50f6d907465456da017ef879b6a11d15c1cfddc76dfc8eb3b6f122532239616) for now.